R/rdeephaven/R/table_handle_wrapper.R

```diff
@@ -2,6 +2,9 @@
 #' @description Deephaven TableHandles are references to tables living on a Deephaven server. They provide an
 #' interface for interacting with tables on the server.
 #' 
+#' @field is_static Indicator variable for whether the table referenced by this TableHandle is static or not.
+#' @field num_rows Number of rows in the table referenced by this TableHandle. This value is only well-defined for static tables.
```

Comment:
I _think_ you are querying the number of rows at construction time and setting that as a constant.  That is never going to give the desired result.  It should be a function that gets the current value.

For API design, there is a principle of least surprise.  You want to minimize how many times you surprise the user -- ideally never.  This result will be very surprising in many cases.  On the other hand, `is_static` is an invariant, so it will never surprise the user.